### PR TITLE
Feature/19 settings: added user configuration setting for time interval

### PR DESCRIPTION
**Pull request by RhoneLachner**

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -53,6 +53,20 @@ const activate = async (context) => {
 		}, 3000)
 	})
 
+	//THIS IS WHERE STOP WATCH FUNCTION BEGINS
```
Comment (jillianlg) [inline extension.js]: A reminder that line 56 - 67 needs to be on a new branch with a corresponding ticket for each ie: "feature/#-commands" or "feature/#-resetTime", etc. however the teams decide to break up those tickets. ;)

File: package.json [hunk 2]
```diff
@@ -23,9 +23,41 @@
       {
         "command": "be-human.trackTime",
         "title": "Time Tracking"
+      },
+      {
+        "command": "be-human.resetTime",
```
Comment (jillianlg) [inline package.json]: Same deal with line 26 - 38. New branch and ticket!

**Comment by jillianlg** [comment]
Could we get the command code moved to a new branch before we push this? 
Similar to how I moved the timeInterval code off of the publish branch to the settings branch.

**Review by jillianlg — CHANGES_REQUESTED** [review]
Could we get the command code moved to a new branch before we push this to dev?
Similar to how I moved the timeInterval code off of the publish branch to the settings branch. :)

We should avoid pushing any "in progress" code to the dev branch.

**Review by bryanakitchen — APPROVED** [review]
Good stuff!

**Review by jillianlg — APPROVED** [review]

**Comment by jillianlg** [comment]
Thank you!!!! APPROVED. :)

**Pull request merged**